lib/levels.js

```diff
@@ -6,17 +6,10 @@ var levels = {
   debug: 3
 };
 module.exports = function (level) {
+  /* istanbul ignore else */
   if (typeof levels[level] === 'number') {
     return levels[level];
-  }
-  if (process.env.NODE_ENV && process.env.NODE_ENV in levels) {
-    return levels[process.env.NODE_ENV];
-  }
-  if (process.env.NODE_ENV === 'production' || process.env.NODE_ENV === 'prod' ) {
-    return 0;
-  }
-  if (typeof process.env.NODE_ENV === 'undefined' ||
-    process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'dev') {
```

Comment:
Ah nevermind, we can keep it. It's not hurting much. I'll add `/* ignore */` flags.